modules/timeutil/since.go

```diff
@@ -23,7 +23,7 @@ const (
 )
 
 func computeTimeDiffFloor(diff int64, lang translation.Locale) (int64, string) {
-	diffStr := ""
+	var diffStr string
```

Comment:
I don't quite see the benefit, but okay?

Reply:
IIRC that's lint rule's quirk .....

Reply:
Previous code was erroring with `assigned to diffStr, but reassigned without using the value`. I'm not sure how else to solve. [Rule here](https://github.com/sanposhiho/wastedassign).